src/popup/components/BackToTop.vue

```diff
@@ -93,8 +93,8 @@ export default defineComponent({
     left: 0;
     width: 100%;
     padding: 12px 4px 4px 0;
-    background: $color-bg-4;
     /* stylelint-disable-next-line declaration-block-no-duplicate-properties */
```

Comment:
This comment can probably also be removed since the duplicate property was removed

Reply:
The property has been moved. The new `stylelint` thinks the duplicate is the first property because the second property is applied.

Reply:
Perhaps we can set it like this: 
`background-color: $color-bg-4;`
and 
` background-image: linear-gradient(...`
This should have the same results without the need to disable stylelint

Reply:
![backToTop](https://github.com/user-attachments/assets/f501dd11-b877-4316-b9fc-b71e353f1e73)